### ISSUE TITLE
bazel test fails locally and on TravisCI

### ISSUE THREAD

**Issue by gregmagolan** [bazelbuild/bazel-watcher#72]
Not sure where to begin to track this down... failure on Travis looked like this on my PR.

```
INFO: Found 14 targets and 5 test targets...
INFO: Building...
[0 / 1] BazelWorkspaceStatusAction stable-status.txt
[2 / 35] Writing file external/org_golang_x_sys/unix/libgo_default_library.cgo\
_c_lib.lo-2.params
[5 / 55] Compiling external/com_github_google_protobuf/src/google/protobuf/com\
piler/java/java_map_field.cc [for host]
[26 / 74] Compiling external/com_github_google_protobuf/src/google/protobuf/em\
pty.pb.cc [for host]
[57 / 105] Compiling external/com_github_google_protobuf/src/google/protobuf/c\
ompiler/java/java_message_field.cc [for host]
[84 / 132] Compiling external/com_github_google_protobuf/src/google/protobuf/u\
til/internal/utility.cc [for host]
[188 / 243] Still waiting for 2 jobs to complete:
      Running (local):
        SkylarkAction external/go_stdlib_linux_amd64_cgo/bin/go [for host], 10\
 s
      Scheduling:
        Linking external/com_github_google_protobuf/protoc [for host], 10 s
[215 / 291] GoCompile external/io_bazel_rules_go/go/tools/builders/~stripped~a\
sm~/github.com/bazelbuild/rules_go/go/tools/builders/asm.a [for host]
[222 / 291] GoCompile e2e/simple/~stripped~sample~/github.com/bazelbuild/bazel\
-watcher/e2e/sample.a
[244 / 291] Compiling external/org_golang_x_sys/unix/go_default_library.cgo.di\
r/external_org_golang_x_sys_unix_syscall_linux.cgo2.c
PASS: //bazel:go_default_test
PASS: //ibazel/command:go_default_test
PASS: //ibazel:go_default_test
PASS: //e2e:go_default_test
INFO: Elapsed time: 242.429s, Critical Path: 42.66s
java.lang.IllegalStateException
	at com.google.common.base.Preconditions.checkState(Preconditions.java:429)
	at com.google.devtools.build.lib.util.Preconditions.checkState(Preconditions.java:196)
	at com.google.devtools.build.lib.runtime.TestResultAnalyzer.differentialAnalyzeAndReport(TestResultAnalyzer.java:115)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.analyzeTestResults(TestCommand.java:172)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.doTest(TestCommand.java:145)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.exec(TestCommand.java:104)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:600)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:351)
	at com.google.devtools.build.lib.runtime.BlazeRuntime.batchMain(BlazeRuntime.java:755)
	at com.google.devtools.build.lib.runtime.BlazeRuntime.main(BlazeRuntime.java:551)
	at com.google.devtools.build.lib.bazel.BazelMain.main(BazelMain.java:57)
java.lang.IllegalStateException
	at com.google.common.base.Preconditions.checkState(Preconditions.java:429)
	at com.google.devtools.build.lib.util.Preconditions.checkState(Preconditions.java:196)
	at com.google.devtools.build.lib.runtime.TestResultAnalyzer.differentialAnalyzeAndReport(TestResultAnalyzer.java:115)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.analyzeTestResults(TestCommand.java:172)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.doTest(TestCommand.java:145)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.exec(TestCommand.java:104)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:600)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:351)
	at com.google.devtools.build.lib.runtime.BlazeRuntime.batchMain(BlazeRuntime.java:755)
	at com.google.devtools.build.lib.runtime.BlazeRuntime.main(BlazeRuntime.java:551)
	at com.google.devtools.build.lib.bazel.BazelMain.main(BazelMain.java:57)
The command "bazel \
  --output_base=$HOME/.cache/bazel \
  --batch \
  --host_jvm_args=-Xmx500m \
  --host_jvm_args=-Xms500m \
  --bazelrc=.test-bazelrc \
  test \
  --experimental_repository_cache="$HOME/.bazel_repository_cache" \
  --local_resources=400,1,1.0 \
  --test_tag_filters=-dev \
  //...
" exited with 37.
```

Locally I can something similar but different using `bazel --bazelrc=.test-bazelrc test --test_tag_filters=-dev ...` on both master and on my live-reload branch:

```
NFO: Found 14 targets and 5 test targets...
INFO: Elapsed time: 6.972s, Critical Path: 0.74s
java.lang.IllegalStateException
	at com.google.common.base.Preconditions.checkState(Preconditions.java:429)
	at com.google.devtools.build.lib.util.Preconditions.checkState(Preconditions.java:196)
	at com.google.devtools.build.lib.runtime.TestResultAnalyzer.differentialAnalyzeAndReport(TestResultAnalyzer.java:115)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.analyzeTestResults(TestCommand.java:176)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.doTest(TestCommand.java:149)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.exec(TestCommand.java:104)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:600)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:351)
	at com.google.devtools.build.lib.runtime.BlazeRuntime.batchMain(BlazeRuntime.java:755)
	at com.google.devtools.build.lib.runtime.BlazeRuntime.main(BlazeRuntime.java:551)
	at com.google.devtools.build.lib.bazel.BazelMain.main(BazelMain.java:57)
java.lang.IllegalStateException
	at com.google.common.base.Preconditions.checkState(Preconditions.java:429)
	at com.google.devtools.build.lib.util.Preconditions.checkState(Preconditions.java:196)
	at com.google.devtools.build.lib.runtime.TestResultAnalyzer.differentialAnalyzeAndReport(TestResultAnalyzer.java:115)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.analyzeTestResults(TestCommand.java:176)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.doTest(TestCommand.java:149)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.exec(TestCommand.java:104)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:600)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:351)
	at com.google.devtools.build.lib.runtime.BlazeRuntime.batchMain(BlazeRuntime.java:755)
	at com.google.devtools.build.lib.runtime.BlazeRuntime.main(BlazeRuntime.java:551)
	at com.google.devtools.build.lib.bazel.BazelMain.main(BazelMain.java:57)
```

Any ideas? I'd like to write a test for the live reload but this is blocking me.

**Comment by gregmagolan** [comment]
@achew22 @alexeagle ^

**Comment by dougkoch** [comment]
This failure is fixed by bazel 0.8.0. The plan is to just bump the minimum bazel version to 0.8.0 once it's released, which should be within the next week or two. 0.8.0-rc2 is available now.

**Comment by gregmagolan** [comment]
Great. Thanks @dougkoch. I'll try 0.8.0-rc2 locally.
